src/Command/GuideTestCommand.php

```diff
@@ -80,4 +87,37 @@ private function getGuideName(string $guide): string
 
         return str_replace('.php', '', end($expl));
     }
+
+    private function getFrontMatter(string $guide): array
+    {
+        $handle = fopen($guide, 'r');
```

Comment:
why so much trouble? We've a library for that in PHP IIRC ? https://github.com/mnapoli/FrontYAML or https://github.com/Modularr/YAML-FrontMatter

Reply:
The front matter in the guides is preceded by php opening commentary ( `//` ) so it is not regularly parsed by usual libraries